packages/app/src/Element/Bookmarks.tsx

```diff
@@ -43,14 +44,21 @@ const Bookmarks = ({ pubkey, bookmarks, related }: BookmarksProps) => {
       {bookmarks
         .filter(b => (onlyPubkey === "all" ? true : b.pubkey === onlyPubkey))
         .map(n => {
-          return (
-            <Note
-              key={n.id}
-              data={n}
-              related={related}
-              options={{ showTime: false, showBookmarked: true, canUnbookmark: loginPubKey === pubkey }}
-            />
-          );
+          switch (n.kind) {
+            case EventKind.Reaction:
+            case EventKind.Repost:
+            case EventKind.ZapReceipt:
+              return <NoteReaction data={n} key={n.id} />;
```

Comment:
I dont think this supports removing the bookmark or showing that its a bookmark like we have with the `Note` kind, would need to support these actions too

Reply:
Got it! Let me work on it